src/js/BookReader/Mode2Up.js

```diff
@@ -108,14 +93,75 @@ export class Mode2Up {
     }
 
     // Prepare view with new center to minimize visual glitches
-    this.prepareTwoPageView(oldCenter.percentageX, oldCenter.percentageY);
+    const drawNewSpread = true;
+    this.prepareTwoPageView(oldCenter.percentageX, oldCenter.percentageY, drawNewSpread);
+  }
+
+  /**
+   *
+   */
+  shouldRedrawSpread() {
+    const { prefetchedImgs, twoPage } = this.br;
+    const { reduce: idealReductionFactor } = this.getIdealSpreadSize( this.br.twoPage.currentIndexL, this.br.twoPage.currentIndexR );
+
+    const currentImagesAreLarger = this.br.reduce <= idealReductionFactor;
+    const pagesInViewArePrefetched = prefetchedImgs[twoPage.currentIndexL] && prefetchedImgs[twoPage.currentIndexR];
+
+    console.log('------ shouldRedrawSpread');
+    console.log('prefetchedImgs', prefetchedImgs);
+    console.log('this.br.reduce', this.br.reduce);
+    console.log('idealReductionFactor', idealReductionFactor);
+    console.log('twoPage', twoPage);
+    console.log("WILL REDRAW: !currentImagesAreLarger || !pagesInViewArePrefetched ", !currentImagesAreLarger || !pagesInViewArePrefetched);
+    console.log('------ end shouldRedrawSpread');
+
+    return !currentImagesAreLarger || !pagesInViewArePrefetched;
+  }
+
+  /**
+   * Resize spread containers, does not prefetch
+   * uses `this.br.twoPage` properties
+   */
+  resizeSpread() {
+    this.br.resizeBRcontainer(false); // no animation
+    this.calculateSpreadSize();
+    const {
+      mainContainerCss,
+      spreadCoverCss,
+      leafEdgeRCss,
+      leafEdgeLCss,
+      spineCss,
+      leftLeafCss,
+      rightLeafCss,
+    } = this.spreadCSSAttributes();
+
+    this.br.refs?.$brTwoPageView.css(mainContainerCss);
+    this.centerView(undefined, undefined); // let function self adjust
+
+    $(this.br.twoPage.coverDiv).css(spreadCoverCss); // click sheath is memoized somehow
+    const $spreadLayers = this.br.refs.$brTwoPageView;
+
+    $spreadLayers.find('.BRleafEdgeR')?.css(leafEdgeRCss);
+    $spreadLayers.find('.BRleafEdgeL')?.css(leafEdgeLCss);
+    $spreadLayers.find('.BRgutter')?.css(spineCss);
+
+    for (const imageIdx in this.br.prefetchedImgs) {
+      const image = this.br.prefetchedImgs[imageIdx];
+      if (image.getAttribute('data-side') === 'L') {
+        $(image).css(leftLeafCss);
+      }
+      if (image.getAttribute('data-side') === 'R') {
+        $(image).css(rightLeafCss);
+      }
+    }
   }
 
   /**
    * @param {number} centerPercentageX
    * @param {number} centerPercentageY
+   * @param {Boolean} drawNewSpread
    */
-  prepareTwoPageView(centerPercentageX, centerPercentageY) {
+  prepareTwoPageView(centerPercentageX = 0.5, centerPercentageY = 0.5, drawNewSpread = false) {
     // Some decisions about two page view:
```

Comment:
new arg to bypass check.  only used in zoom mode where we actually need a redraw